README.md

```diff
@@ -1231,4 +1231,4 @@ You can run Tyk Pump in demo mode, which will generate fake analytics data and s
 - `--demo-api=<API_ID>` - Sets the API ID to use for the demo data. If not set, a random API ID will be generated.
```

Comment:
I think this means that this <API_ID> will be recorded in the demo transactions.
I would re-word this slightly to:
`Configure the value to be recorded as the <API_ID> in the demo transactions. If this option is not set, the Pump Demo mode will use a random <API_ID>. Note that the same <API_ID> will be used for all transaction logs.`